examples/medplum-provider/src/components/tasks/TaskBoard.tsx

```diff
@@ -140,14 +139,12 @@ export function TaskBoard(props: TaskBoardProps): JSX.Element {
     handleTaskChange(task).catch(showErrorNotification);
   };
 
-  const handleTaskChange = async (task: Task): Promise<void> => {
-    setTasks(tasks.map((t) => (t.id === task.id ? task : t)));
-    onTaskChange(task);
-    setSelectedTask(task);
+  const handleTaskChange = async (_task: Task): Promise<void> => {
+    await fetchTasks().catch(showErrorNotification);
   };
 
   const handleDeleteTask = async (task: Task): Promise<void> => {
-    setTasks(tasks.filter((t) => t.id !== task.id));
+    await fetchTasks().catch(showErrorNotification);
```

Comment:
It looks like `showErrorNotification` is always used as an error handler for `fetchTasks`, does it make sense to move that into that handler?